app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  before_action :redirect_to_home
+
+  def new
+    @user = User.new
```

Comment:
Ideally this action should not be accessed by logged in users. You can do `redirect_to some_path and return if logged_in?`